test/suites/integration/suite_test.go

```diff
@@ -20,14 +23,30 @@ import (
 )
 
 const (
-	k8snamespace = "e2e-test"
+	k8snamespace                = "e2e-test"
+	secondaryAccountTestRoleArn = "SECONDARY_ACCOUNT_TEST_ROLE_ARN"
 )
 
+var k8sTestTags = map[string]*string{
+	anaws.TagBase + "TestSuite": aws.String("sharing"),
```

Comment:
Do you need to define `TestSuite:sharing` tag in the `ram_share_test.go` file? or change the tag value?

Reply:
Ah good point, this should just be the e2e test suite